css-color-4/Overview.bs

```diff
@@ -614,89 +614,151 @@ Transparency: the 'opacity' property</h2>
+	<dfn export>transparent black</dfn> is the same color,
+	but fully transparent--
+	i.e. <nobr>''rgb(0 0 0 / 0%)''</nobr>.
+
+<h3 id="color-syntax">The <<color>> syntax</h3>
 
 	Colors in CSS are represented by the <dfn export><<color>></dfn> type:
 
 	<pre class='prod'>
 	&lt;color> = <<absolute-color-base>> | currentcolor | <<system-color>> | <<device-cmyk()>>
 
-	<dfn>&lt;absolute-color-base></dfn> = <<hex-color>> | <<named-color>> | transparent |
-	    <<rgb()>> | <<rgba()>> | <<hsl()>> | <<hsla()>> | <<hwb()>> |
-	    <<lab()>> | <<lch()>> |
-	    <<oklab()>> | <<oklch()>> |
-	    <<color()>>
+	<dfn>&lt;absolute-color-base></dfn> = <<hex-color>> | <<absolute-color-function>> | <<named-color>> | transparent
+	<dfn>&lt;absolute-color-function></dfn> = <<rgb()>> | <<rgba()>> |
+	                            <<hsl()>> | <<hsla()>> | <<hwb()>> |
+	                            <<lab()>> | <<lch()>> | <<oklab()>> | <<oklch()>> |
+	                            <<color()>>
 	</pre>
 
-	The <dfn export>color-functions</dfn> are <<rgb()>>, <<rgba()>>,
-		<<hsl()>>, <<hsla()>>, <<hwb()>>,
-		<<lab()>>, <<lch()>>,
-		<<oklab()>>, <<oklch()>>, and
-		<<color()>>.
-
-	Of those, <<hsl()>>, <<hsla()>>, <<hwb()>>, <<lch()>>, and <<oklch()>>
-		are <dfn export>cylindrical polar color</dfn> representations,
-		which specify color using a <<hue>> angle,
-		a central axis representing lightness
-		(black-to-white),
-		and a radius representing saturation or chroma
-		(how far the color is from a neutral grey).
-	The other color spaces are
-		<dfn export>rectangular orthogonal color</dfn> representations,
-		which specify color using three
-		(or more, in the case of CMYK or CMYKOGV)
-		orthogonal component axes.
+	The <<hsl()>>, <<hsla()>>, <<hwb()>>, <<lch()>>, and <<oklch()>> [=color functions=]
+	are [=cylindrical polar color=] representations using a <<hue>> angle;
+	the other [=color functions=] use [=rectangular orthogonal color=] representations.
 
-	For easy reference in other specifications,
-	<dfn export>opaque black</dfn> is defined as the color <nobr>''rgb(0 0 0 / 100%)''</nobr>;
-	<dfn export>transparent black</dfn> is the same color,
-	but fully transparent--
-	i.e. <nobr>''rgb(0 0 0 / 0%)''</nobr>.
-
-	All of the <<color>> syntactic forms
+	All of the <<absolute-color-function>> syntactic forms
 	defined in this specification
 	use space, not comma,
 	to separate the color components.
 	They also use a solidus ("/")
 	to separate an optional alpha term
 	from the color components.
 
+	<div class=example id="example-generic-syntax">
+		<p>The following represents a saturated sRGB red that is 50% opaque:
+		<pre>rgb(100% 0% 0% / 50%)</pre>
+	</div>
+
+<h4 id="color-syntax-legacy">
+Legacy Comma-separated Color Function Syntax</h4>
+
 	For Web compatibility,
-	<<color>> syntactic forms
-	defined in earlier levels of this specification
-	also support
+	the syntactic forms
+	of ''rgb()'', ''rgba()'', ''hsl()'', ''hsla()'', and ''device-cmyk()''
```

Comment:
device-cmyk is not in Color 4 (and is not legacy, was not defined in Color 3)

Reply:
It's legacy because it's implemented with commas in both PrinceXML and Antenna House, and possibly some other CSS-to-PDF implementations, so there's a compat issue there. (It was specced originally in GCPM, which didn't get very far on the REC track, but there were a number of features that were implemented and deployed in the print world.)

Commented out for now, but, I think we need to keep it as part of this section once it's re-introduced.

Reply:
I believe they called it `cmyk()` though

Reply:
Added https://github.com/w3c/csswg-drafts/commit/475eb97836cc101b9c569e5c218caffd5e778e4e